lib/UserSearch/styles.scss

```diff
@@ -29,14 +29,14 @@
   .toggle-wrapper {
     margin-left: auto;
     flex-shrink: 0;
-    padding-right: $layout-spacing-base * .75;
+    padding-right: $layout-spacing-base * 0.75;
```

Comment:
I wonder why we `* 0.75` in the first place 🤔 

Is this padding used anywhere else in `gather-ui`? If not I recommend we fix this to make it more consistent with every other UI component 👍